example-utils/integration-test-utils/src/integration_test_utils.rs

```diff
@@ -0,0 +1,335 @@
+                            println!("{name} had rx of: {times}");
+                            task::sleep(Duration::from_millis(1000)).await;
+                            return sent_messages;
+                        } else {
+                            match &*command {
+                                ClientCommand::NoOp => {}
+                                ClientCommand::ConnectedToStreamer(active_connections) => {
+                                    debug!("{} commmand: ConnectedToStreamer", &name);
+                                    active_connection_listing = active_connections.clone();
+                                    debug!(
+                                        "{} set connected_to_streamer to: {:?}",
+                                        &name, active_connection_listing
+                                    );
+                                }
+                                ClientCommand::DisconnectedFromStreamer(active_connections) => {
+                                    debug!("{} commmand: DisconnectedFromStreamer", &name);
+                                    active_connection_listing = active_connections.clone();
+                                    debug!(
+                                        "{} set connected_to_streamer to: {:?}",
+                                        &name, active_connection_listing
+                                    );
+                                }
+                                _ => {
+                                    error!(
+                                        "{} ClientCommand::Stop should have been handled earlier",
+                                        &name
+                                    )
+                                }
+                            }
+                            {
+                                let (lock_exec_pause, cvar_exec_pause) = &*execution_paused;
+                                let mut has_paused = lock_exec_pause.lock().await;
+                                *has_paused = true;
+                                debug!("{} has_paused set to true", &name);
+                                cvar_exec_pause.notify_one();
+                                debug!("{} cvar_exec_pause.notify_one()", &name);
+                            }
+                            debug!("{} Loop paused. Waiting...", &name);
+                            should_pause = cvar.wait(should_pause).await;
+                            debug!("{} Got signal to pause", &name);
+                        }
+                    }
+                }
+
+                let current = Instant::now();
+                let ellapsed = current - start;
+
+                debug!("ellapsed: {}", ellapsed.as_millis());
+
+                debug!("-----------------------------------------------------------------------");
+
+                if !send {
+                    continue;
+                }
+
+                // TODO: Doesn't work currently
```

Comment:
For same reasoning as [here](https://github.com/eclipse-uprotocol/up-streamer-rust/pull/8/files/a53b62db291a2e74dd6b5022863b802db36a2bc5#r1582008874) this should also be removed.

Reply:
Done